### ISSUE TITLE
多卡推理的问题,4张2080ti可以多卡推理，但是两张5880设置cuda为auto就会失败甚至直接关机，单张5880推理没有问题

### ISSUE THREAD

**Issue by dyhuachi** [zai-org/GLM-V#192]
### System Info / 系統信息

软件版本均为requirenment版本

### Information / 问题信息

- [x] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

import argparse
import re
import torch
from transformers import AutoProcessor, Glm4vForConditionalGeneration

def build_content(image_paths, video_path, text):
    content = []
    if image_paths:
        for img_path in image_paths:
            content.append({"type": "image", "url": img_path})
    if video_path:
        content.append({"type": "video", "url": video_path})
    content.append({"type": "text", "text": text})
    return content

processor = AutoProcessor.from_pretrained("/root/autodl-tmp/ZhipuAI/GLM-4.1V-9B-Thinking", use_fast=True)
model = Glm4vForConditionalGeneration.from_pretrained(
    "/root/autodl-tmp/ZhipuAI/GLM-4.1V-9B-Thinking", torch_dtype=torch.bfloat16, device_map="cuda:0"
)
messages = []
messages.append(
    {
        "role": "user",
        "content": [
            {"type": "video", "url": "/root/autodl-tmp/media/demo3.mp4"},
            {"type": "text", "text": "这个视频的内容是什么？"},
        ],
    }
)
inputs = processor.apply_chat_template(
    messages,
    tokenize=True,
    add_generation_prompt=True,
    return_dict=True,
    return_tensors="pt",
    padding=True,
).to(model.device)

output = model.generate(
    **inputs,
    max_new_tokens=1024,
    repetition_penalty=1.0,
    do_sample=1.0 > 0,
    top_k=2,
    temperature=1.0,
)
raw = processor.decode(
    output[0][inputs["input_ids"].shape[1] : -1], skip_special_tokens=False
)
match = re.search(r"<answer>(.*?)</answer>", raw, re.DOTALL)
answer = match.group(1).strip() if match else ""
messages.append(
    {"role": "assistant", "content": [{"type": "text", "text": answer}]}
)
print(f"Assistant: {raw}")

### Expected behavior / 期待表现

1.如果知道原因可以反馈给我
2.我的推理速度和调用官方api的速度完全不一致，要慢很多倍，是参数问题吗，是否可以帮助一下；测试本地部署的提示词和调用api的一致

**Comment by zRzRzRzRzRzRzR** [comment]
官方API和开源速度不一样是正常的， 官方API是有优化和加速（具体细节不方便透露）。应该比sglang在h100集群的速度都快一点。
另外，没有测试过这卡的数据，暂时不知道为什么，仅测试H100 和 B200 正常使用
